artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/messages/DestAbstract.java

```diff
@@ -21,29 +21,29 @@
 import javax.jms.JMSException;
 import javax.jms.Session;
 
-import com.github.rvesse.airline.annotations.Option;
 import org.apache.activemq.artemis.cli.factory.serialize.MessageSerializer;
 import org.apache.activemq.artemis.cli.factory.serialize.XMLMessageSerializer;
 import org.apache.activemq.artemis.jms.client.ActiveMQDestination;
+import picocli.CommandLine.Option;
 
 public class DestAbstract extends ConnectionAbstract {
 
-   @Option(name = "--destination", description = "Destination to be used. It can be prefixed with queue:// or topic:// and can be an FQQN in the form of <address>::<queue>. Default: queue://TEST.")
+   @Option(names = "--destination", description = "Destination to be used. It can be prefixed with queue:// or topic:// and can be an FQQN in the form of <address>::<queue>. Default: queue://TEST.")
    String destination = "queue://TEST";
 
-   @Option(name = "--message-count", description = "Number of messages to act on. Default: 1000.")
+   @Option(names = "--message-count", description = "Number of messages to act on. Default: 1000.")
    int messageCount = 1000;
 
-   @Option(name = "--sleep", description = "Time wait between each message.")
+   @Option(names = "--sleep", description = "Time wait between each message.")
    int sleep = 0;
 
-   @Option(name = "--txt-size", description = "Transaction batch size.")
+   @Option(names = {"--txt-size", "--tx-batch-size"}, description = "Transaction batch size. (--txt-size is deprecated)")
```

Comment:
The non-deprecated option name being first would read better (unless the cli is reverse-ordering the options)

Reply:
I'm using hidden now...


and I opened a PR and bug on pico to fix the auto-completion on the shell:

https://github.com/remkop/picocli/pull/2075